### PR TITLE
Move contributions section

### PR DESCRIPTION
**Pull request by tpdorsey**
## Description

Move body of the contributions instructions to later in the readme, per feedback.

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## New Recipe Checklist

- [ ] My PR is in the format of `Add <project-name> recipe`
- [ ] My recipe is reproducible and explains everything needed to run successfully. If my code has external dependencies, I make mention of them.
- [ ] My code is easily understandable and/or well-commented.
- [ ] If my recipe requires a new category (e.g. creating a monitoring/ folder in devops/), I have encapsulated my project within its own subfolder so that others can add their own recipes as well.
- [ ] If my recipe uses Prefect < 2.0, my code is within the prefect-v1-legacy/ folder.

### CODE REVIEW

**Comment by ahuang11** [comment]
This looks good.

Another suggestion is hyperlinking Table of Contents in the intro?

**Review by robfreedy — DISMISSED** [review]

**Review by serinamarie — CHANGES_REQUESTED** [review]
Would you mind adding GitHub Actions under Diving Deeper in the Table of Contents? Could you also hyperlink 'our recipes' to the Table of Contents?

File: README.md [hunk 1]
```diff
@@ -6,27 +6,12 @@
 
 # Prefect Recipes 🧑‍🍳 🥐
 
-This repository contains common and extensible Prefect patterns to drive efficient workflows &mdash; we like to call these patterns our **recipes**!
+This repository contains common and extensible Prefect patterns to drive efficient workflows &mdash; we like to call these patterns [our **recipes**](#table-of-contents-📖)!
```
Comment (serinamarie) [inline README.md:9]: I think you'll need to remove the book emoji for the link to work, but keep the dash after contents

i.e. `[our **recipes**](#table-of-contents-)!`

Comment (tpdorsey) [reply README.md]: Ah, annoying that VSC preview does not recognize links the same way GitHub does. Fixing...

**Review by serinamarie — APPROVED** [review]

**Pull request merged**